scripts/west_commands/runners/probe_rs.py

```diff
@@ -5,12 +5,14 @@
 
 from runners.core import RunnerCaps, ZephyrBinaryRunner
 
+DEFAULT_PROBE_RS_GDB_CONNECTION_STRING = 'localhost:1337'
```

Comment:
Typically, it's split into configurable `host` and `port` properties.

```suggestion
DEFAULT_PROBE_RS_GDB_HOST = 'localhost'
DEFAULT_PROBE_RS_GDB_PORT = 1337
```